typescript/src/electrum.ts

```diff
@@ -253,6 +253,58 @@ export class Client implements BitcoinClient {
     )
   }
 
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   * @see {BitcoinClient#getTransactionHistory}
+   */
+  getTransactionHistory(
+    address: string,
+    limit?: number
+  ): Promise<Transaction[]> {
+    return this.withElectrum<Transaction[]>(async (electrum: Electrum) => {
+      const script = bcoin.Script.fromAddress(address).toRaw().toString("hex")
```

Comment:
Can't we use `createOutputScriptFromAddress` function?

Reply:
Good catch! See 056cac5b